11-Backbonejs/Adding-Custom-Events.md

```diff
@@ -34,44 +34,42 @@ Remember [Ruby Mix-ins](https://www.tutorialspoint.com/ruby/ruby_modules.htm)?
 ```javascript
 // generic Javascript example
 
-var task = {
+const task = {
   assignee: "Dan",
-  task:     "Do work",
+  task_name: "Accomplish a thing",
   due_date: "01/01/2017"
 };
 
 _.extend(task, Backbone.Events);
 
-task.listenTo(task, "Past_Due", function(message) {
-    $("#content").append("<h2>Your Task <span class='red'>" + this.task + 
-    	"</span> is " + message.msg + ".</h2> <p> Current Date: " +
-    	 message.date.toLocaleDateString('en-US') + "</p>")
+task.listenTo(task, "past_due", function(message) {
+  $("#content").append(`<h2>Your task <span class='red'> ${this.task_name} </span> is ${message.msg}.</h2> <p> Current Date: ${message.date.toLocaleDateString('en-US')} </p>`)
 });
 
-task.trigger("Past_Due", {msg: "past due",
-							date: new Date()						
+task.trigger("past_due", {
+  msg: "past due",
+  date: new Date(),
 });
 ```
 
-You can play with this in action [here:](http://codepen.io/CheezItMan/pen/JbyrOG).
+You can play with this in action [here](https://codepen.io/adadev/pen/QOROxX).
```

Comment:
Fixed the bug!

I like the click because it console.logs the `task_name` property of the model associated with that specific view that was clicked on, but I'll think on how to make it more interesting/meaningful